crates/database/src/orders.rs

```diff
@@ -638,6 +638,11 @@ pub fn user_orders<'a>(
 " LEFT OUTER JOIN onchain_placed_orders onchain_o on onchain_o.uid = o.uid",
 " WHERE onchain_o.sender = $1 ",
 " ORDER BY creation_timestamp DESC LIMIT $2 + $3 ) ",
+" UNION ",
+" (SELECT ", crate::jit_orders::JIT_ORDERS_SELECT,
+" FROM jit_orders o",
+" WHERE o.owner = $1 AND NOT EXISTS (SELECT 1 FROM orders ord WHERE o.uid = ord.uid)",
+" ORDER BY creation_timestamp DESC LIMIT $2 + $3 ) ",
```

Comment:
Do we really need this while we already have global LIMIT/OFFSET?
```suggestion
" ORDER BY creation_timestamp DESC ) ",
```